packages/chain-mon/src/fault-mon/service.ts

```diff
@@ -148,7 +148,7 @@ export class FaultDetector extends BaseServiceV2<Options, Metrics, State> {
 
       this.logger.info('fetching L2OutputOracle contract from OptimismPortal')
       const opts = {
-        portalAddress,
+        address: portalAddress,
         signerOrProvider: this.options.l1RpcProvider,
       }
       const portalContract = getOEContract('OptimismPortal', l2ChainId, opts)
```

Comment:
could consider inlining opts here so that typescript would have caught this issue